### ISSUE TITLE
Says "AppImage Added", but that doesn't seem to be the case

### ISSUE THREAD

**Issue by SeaGull711** [probonopd/go-appimage#187]
I'm running a fresh install of Ubuntu 21.10, I downloaded AppImaged (-650-aarch64.AppImage)  and, as soon as the download completed, I got a notification that said, "AppImage Added" (Paraphrasing) and then I moved it to ~/.local/bin/ and I got a notification saying, "AppImage Removed" and then "AppImage Added". The same thing happens with any other AppImages I'm downloading and moving, BUT then when I try to find it in the app search, these Apps don't come up.  I made sure they're set to executable, and if I run them, they seem to work, but I thought they were supposed to show up in the App Menu... isn't that the point?

Any advice is appreciated!

**Comment by probonopd** [comment]
Yes, they should show up in the menu.
Try logging out from your desktop and logging in again.
Also try without moving the AppImages around.

**Comment by SeaGull711** [comment]
OK, so I rebooted and downloaded another AppImage, but got no "AppImage added" notification when the download completed.  I tried moving it to ~/.local/bin/ again and got no removed/added notifications.  So I went to ~/.local/bin/ and deleted appimaged-650-aarch64.AppImage and re-downloaded it.  This time, no notification when that download completed either, or upon moving it even though I got them the first time.  😕

**Comment by SeaGull711** [comment]
I'm an idiot.  I first tried the x86_64 version and mistakenly read the notification, "You're not running one of the approved live versions...." as an error message, so I figured I had the architecture wrong.  Then I tried the aarch64 version, but in reality, it was the first version still working (kinda) until I rebooted.  I just reinstalled the x86_64 version and all is working as it should.
Sorry f
